app/src/main/java/com/infomaniak/drive/ui/fileList/SearchFragment.kt

```diff
@@ -214,7 +225,7 @@ class SearchFragment : FileListFragment() {
 
             val oldList = fileListViewModel.oldList?.toMutableList() as? ArrayList
```

Comment:
Now the oldList becomes useless

Reply:
Now it is used, to avoid another type of bug that can arise